pkg/api/handlers/libpod/play.go

```diff
@@ -105,12 +114,19 @@ func PlayKube(w http.ResponseWriter, r *http.Request) {
 	if _, found := r.URL.Query()["start"]; found {
 		options.Start = types.NewOptionalBool(query.Start)
 	}
-
+	if query.Down {
+		report, err := containerEngine.PlayKubeDown(r.Context(), tmpfile.Name(), options)
+		if err != nil {
+			utils.Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Wrap(err, "error tearing down YAML file"))
+			return
+		}
+		utils.WriteResponse(w, http.StatusOK, report)
+		return
+	}
 	report, err := containerEngine.PlayKube(r.Context(), tmpfile.Name(), options)
 	if err != nil {
 		utils.Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Wrap(err, "error playing YAML file"))
```

Comment:
Does err provide context on which pod and yaml file failed? If not, please add to the wrapped error message for debugging.

Reply:
I believe so